Sources/TUSKit/TUSAPI.swift

```diff
@@ -92,25 +92,25 @@ final class TUSAPI {
     }
     
     func makeCreateRequest(metaData: UploadMetadata) -> URLRequest {
-        func makeMetadataHeaders() -> [String: String] {
-            let fileName = metaData.filePath.lastPathComponent
+        func makeUploadMetaHeader() -> [String: String] {
+            var metaDataDict: [String: String] = [:]
             
-            var metaDataHeaders = metaData.customHeaders ?? [:]
+            let fileName = metaData.filePath.lastPathComponent
             if !fileName.isEmpty && fileName != "/" { // A filename can be invalid, e.g. "/"
-                metaDataHeaders["filename"] = fileName
+                metaDataDict["filename"] = fileName
             }
             
             if let mimeType = metaData.mimeType, !mimeType.isEmpty {
-                metaDataHeaders["filetype"] = mimeType
+                metaDataDict["filetype"] = mimeType
             }
-            return metaDataHeaders
+            return metaDataDict
         }
        
         /// Turn dict into a comma separated base64 string
-        func encode(headers: [String: String]) -> String? {
-            guard !headers.isEmpty else { return nil }
+        func encode(_ dict: [String: String]) -> String? {
```

Comment:
Renamed the attribute here, because it's not operating on headers, it's operating on a dictionary. (Which is the value of a single header)